matrix-sdk-android/src/rustCrypto/java/org/matrix/android/sdk/internal/crypto/CryptoModule.kt

```diff
@@ -246,7 +246,7 @@ internal abstract class CryptoModule {
     abstract fun bindVerificationService(service: RustVerificationService): VerificationService
 
     @Binds
-    abstract fun bindCryptoStore(store: RealmCryptoStore): IMXCryptoStore
+    abstract fun bindCryptoStore(store: RustCryptoStore): IMXCommonCryptoStore
```

Comment:
This is the location of the fix, right?

Reply:
Mainly yes.
The IMXCryptoStore, RealmCryptoStore, InboundGroupSessionStore, OlmSessionStore have all moved to the kotlin flavour source folder.
And there is the new RustCryptoStore in the rust flavour sources